packages/notices/CHANGELOG.md

```diff
@@ -1,8 +1,13 @@
+## 2.0.0 (Unreleased)
+
+### Breaking Change
+
+- Revert: The `speak` option added in v1.1.0 has been removed.
+
 ## 1.1.0 (2018-11-20)
 
 ### New Feature
 
-- The `createNotice` can now optionally accept a WPNotice object as the sole argument.
```

Comment:
This never actually went through in #11604. See https://github.com/WordPress/gutenberg/pull/11604#issuecomment-439982014 .